pkg/commands/git_commands/file.go

```diff
@@ -164,3 +165,13 @@ func (self *FileCommands) guessDefaultEditor() string {
 
 	return editor
 }
+
+func (self *FileCommands) OpenDiffToolCmdObj(fromCommit string, toCommit string, reverse bool, filename string, isDirectory bool) oscommands.ICmdObj {
```

Comment:
I wonder if this would be better as an "options" struct to pass instead of having 5 arguments.

Even if not so, I'd order them as `filename, fromCommit, toCommit, reverse, isDirectory`

Reply:
Good point. Actually the two OpenDiffToolCmdObj functions that I added are so similar that it's worth consolidating them into one, I did that in 3b04c81e60, with an options struct.

Gonna squash now, that's probably easier to review.